packages/flame/test/components/text_box_component_test.dart

```diff
@@ -148,6 +148,20 @@ void main() {
       goldenFile: '../_goldens/text_box_component_test_1.png',
     );
   });
+
+  testGolden(
+    'Big upscale',
+    (game) async {
+      game.addAll([
+        TextBoxComponent(
+          text: 'quickly',
+          pixelRatio: 8,
```

Comment:
this was more than enough to cause issues to me.
on my application anything more than 2.51 was causing issues.
the test seems to be a little more lenient - it appears to be dependent on the actual size of the canvas